connector/oidc/oidc_test.go

```diff
@@ -288,6 +289,66 @@ func TestHandleCallback(t *testing.T) {
 				"email_verified": true,
 			},
 		},
+		{
+			name:               "concatinateClaim",
+			userIDKey:          "", // not configured
+			userNameKey:        "", // not configured
+			expectUserID:       "subvalue",
+			expectUserName:     "namevalue",
+			expectGroups:       []string{"group1", "gh::acme::pipeline-one", "tfe-acme-foobar", "bk-emailvalue"},
+			expectedEmailField: "emailvalue",
+			claimConcatenations: []ClaimConcatenation{
```

Comment:
Can you please add a comment above each case which behavior it tests so that they don't get accidentally refactored?